### PR TITLE
Awesome books

### PR DESCRIPTION
**Pull request by OpondoG**
Awesome books is a simple website that displays a list of books and allows you to add and remove books from that list.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { DateTime } from './module/luxon.js';
+import { deleteRow } from './module/deleteRow.js';
+import { addnewRow } from './module/addNewRow.js';
+import { removeAll } from './module/removeAll.js';
+import { printLocalStorage } from './module/printLocalStorage.js';
+import { addForm, addListmenu, contact } from './module/desktopVersion.js';
+
+// PRINTING THE CURRENT DATE
```
Comment (elmar8287) [inline index.js]: - [x] Please, consider removing all comments to make your code clear. You code enough understandable, so it talks itself about what is going on in your project 👍 
- [x] Please, consider this for all your js files 💯

File: module/addNewRow.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  const books = new Books();
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  const storedBooksList = JSON.parse(localStorage.getItem('books'));
+  books.add(title, author);
+  if (localStorage.getItem('books') === null) {
+    localStorage.setItem('books', JSON.stringify(books.list));
+    /* eslint-disable */
+      location.reload();
+      /* eslint-enable */
```
Comment (elmar8287) [inline module/addNewRow.js]: - [x] I see here many linter disables inserted. Please, consider removing them all from all js files. Let linter makes its direct job to help you to find the bugs in the early stage of development 👍

File: style.css [hunk 3]
```diff
@@ -0,0 +1,118 @@
+.desktop-heading {
```
Comment (elmar8287) [inline style.css]: - [x] Please, consider renaming this `style.css` file to `index.css` - yes, this is not a big issue, but it is required for this milestone. Thanks 👍

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @OpondoG.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good README and PR description
- Nice implementation of design 👍 
- You considered ES6 syntax - well done 👍 

## Required Changes ♻️

- [x] Check the comments under the review.

## Optional suggestions

- Please, consider fixing the broken link to the MIT.md file in your README file 👍 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @OpondoG, 

## Status: APPROVED 🥳 🥳 

Great job working on the required changes 👏  Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**